Ruddarr/Models/Movies/Movie.swift

```diff
@@ -108,7 +113,7 @@ struct Movie: Identifiable, Equatable, Codable {
             return "Waiting"
         }
 
-        if monitored && isReleased {
+        if monitored {
```

Comment:
Fixes bug with the wrong `stateLabel` being returned when a movie is monitored but not yet released.